src/themes/five/src/components/ProductItemAccordion/index.js

```diff
@@ -206,6 +206,10 @@ export const ProductItemAccordion = (props) => {
             )}
           </ProductInfo>
 
+          {isConfirmationPage && (productInfo().ingredients.length > 0 || productInfo().options.length > 0 || product.comment) && (
```

Comment:
rename the var with a name more general, like `showArrowDown` or something like that.

Reply:
updated